laborator/content/reprezentare-numere/solution/2-len_xor/Makefile

```diff
@@ -0,0 +1,2 @@
+PROGNAME := len_xor
+include ../utils/Makefile.generic
```

Comment:
Use the makefiles from the `utils/` folder currently present in the repo: https://github.com/systems-cs-pub-ro/iocla/tree/master/laborator/content/utils.